packages/commons/src/templating/htmlTemplateService.ts

```diff
@@ -12,7 +12,9 @@ export function buildHTMLTemplateService(): HtmlTemplateService {
       context: Record<string, unknown>
     ): string => {
       try {
-        const compileHtml = Handlebars.compile(htmlTemplate);
+        const compileHtml = Handlebars.compile(htmlTemplate, {
+          noEscape: true,
```

Comment:
The `noEscape` flag is important because with the risk anlaysis pdf generation we are interpolating html, and we need to not escape.

Could this be a safety issue since a malevolent user could compile the riskanalysis with a script tag writing js code?
@galales @beetlecrunch

Reply:
this is definitely a red flag, thanks for raising the point!
We could escape the values before interpolating 🤔 , do you see better alternatives?